src/main/java/com/twilio/oai/resolver/node/NodeParameterResolver.java

```diff
@@ -27,9 +27,7 @@ public CodegenParameter resolve(final CodegenParameter codegenParameter) {
             codegenParameter.vendorExtensions.put("x-transform", transform);
         }
 
-        if (mapper.libraries().getMap(codegenParameter.dataFormat).isPresent()){
-            codegenParameter.vendorExtensions.put("x-import", mapper.libraries().get(codegenParameter.dataFormat).orElseThrow());
-        }
+        mapper.libraries().getMap(codegenParameter.dataFormat).ifPresent(imports -> codegenParameter.vendorExtensions.put("x-import", mapper.libraries().get(codegenParameter.dataFormat).orElseThrow()));
```

Comment:
The reason for switching to `ifPresent` is it gives you the value from the map. This means you don't have to `get` the value again. E.g., 
```ts
mapper.libraries().getMap(codegenParameter.dataFormat).ifPresent(imports -> codegenParameter.vendorExtensions.put("x-import", imports));
```